src/components/Trade/services/useTradeQuoteService.ts

```diff
@@ -0,0 +1,156 @@
+import { skipToken } from '@reduxjs/toolkit/query'
+import { ChainId, ethAssetId, fromAssetId } from '@shapeshiftoss/caip'
+import { UtxoBaseAdapter } from '@shapeshiftoss/chain-adapters'
+import { GetTradeQuoteInput, TradeQuote, UtxoSupportedChainIds } from '@shapeshiftoss/swapper'
+import { KnownChainIds } from '@shapeshiftoss/types'
+import { useEffect, useState } from 'react'
+import { useFormContext, useWatch } from 'react-hook-form'
+import { useSelector } from 'react-redux'
+import {
+  getBestSwapperFromArgs,
+  getFirstReceiveAddress,
+  getFormFees,
+  getUtxoParams,
+  isSupportedNoneUtxoSwappingChain,
+  isSupportedUtxoSwappingChain,
+  TradeQuoteInputCommonArgs,
+} from 'components/Trade/hooks/useSwapper/utils'
+import { TradeState } from 'components/Trade/types'
+import { getChainAdapterManager } from 'context/PluginProvider/chainAdapterSingleton'
+import { useWallet } from 'hooks/useWallet/useWallet'
+import { toBaseUnit } from 'lib/math'
+import { useGetTradeQuoteQuery } from 'state/apis/swapper/swapperApi'
+import { selectAccountSpecifiers } from 'state/slices/accountSpecifiersSlice/selectors'
```

Comment:
let's move away from this selector and query this from the portfolioSlice so we can kill the accountSpecifiersSlice entirely

Reply:
Roger that. I'll sort it out in another PR soon.